src/controllers/users.controller.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const status = require('../utils/constants');
+const userService = require('../services/users.services');
+const idGenerator = require('../utils/idGenerator');
+
+const getAll = (req, res) => {
+  res.send(userService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getById(+id);
```

Comment:
```suggestion
  const user = userService.getById(Number(id));
```

Better to use Number it's easier for reading